<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="show_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recognize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>识别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输入文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_message">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addto_db">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加到数据库</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_db_message">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play_audio">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>input_file</tabstop>
  <tabstop>show_file</tabstop>
  <tabstop>addto_db</tabstop>
  <tabstop>recognize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>recognize</sender>
   <signal>clicked()</signal>
   <receiver>show_message</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_file</sender>
   <signal>clicked()</signal>
   <receiver>show_file</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addto_db</sender>
   <signal>clicked()</signal>
   <receiver>show_db_message</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>play_audio</sender>
   <signal>clicked()</signal>
   <receiver>play_audio</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
